src/ctype/CTypeUtils.ts

```diff
@@ -40,12 +40,26 @@ export function verifyClaimStructure(claim: any, schema: any): boolean {
 }
 
 export function getHashForSchema(schema: ICType['schema']): string {
-  return Crypto.hashObjectAsStr(schema)
+  const { $id, ...schemaWithoutID } = schema
```

Comment:
I think that removing the `$id` could be more explicit. there would also be no `'$id' is assigned a value but never used.` Warning.

some thing like this perhaps?
```typescript
  const { ...schemaClone } = schema
  delete schemaClone.$id
  return Crypto.hashObjectAsStr(schemaClone)
```

Reply:
I have updated, it would make it look cleaner. Maybe it is something we might not want to use.